docs/scripts/make_dataset_zoo_docs.py

```
@@ -0,0 +1,182 @@
+
+
+def _normalize_slug(slug: str) -> str:
+    return slug.replace("-", "_")
+
+
+def _extract_table_tags(content: str) -> Dict[str, str]:
+    tags_map = {}
+    pattern = r"\| :ref:`([^<]+) <dataset-zoo-([^>]+)>`\s*\|\s*([^|]+)\s*\|"
+
+    for match in re.finditer(pattern, content, re.MULTILINE):
+        name = match.group(1).strip()
```

Ruff (0.13.3)

Local variable `name` is assigned to but never used

Remove assignment to unused variable `name`

(F841)